src/yaml/mapping.cr

```diff
@@ -29,16 +29,15 @@ module YAML
   #
   # ```
   # employee = Employee.from_yaml("title: Manager\nname: John\nage: 30")
-  # employee.age # => undefined method 'age'.
+  # employee.age # undefined method 'age'. (compile error)
   #
-  # employee = Employee.from_yaml("title: Manager")
-  # # => ParseException: missing yaml attribute: name
+  # Employee.from_yaml("title: Manager") # raises YAML::ParseException
   # ```
   #
   # You can also define attributes for each property.
   #
   # ```
-  # class Employee
+  # class Employer
```

Comment:
Yup, I tried it first. But the approach failed because many examples implicitly depends previous ones like this.
- 1st example: defining `enum Color`
- 2nd example: `Color.green ...`
- 3rd example: `typeof(Color) ...`

I think the 50-80% examples would become failure in my experience.